vcr/api/v2/api.go

```diff
@@ -87,6 +86,14 @@ func (w Wrapper) IssueVC(ctx echo.Context) error {
 		public = *issueRequest.Visibility == IssueVCRequestVisibilityPublic
 	}
 
+	if issueRequest.Type == "" {
+		return core.InvalidInputError("missing credential type")
+	}
+
+	if issueRequest.CredentialSubject == nil {
```

Comment:
what if the element is present, but is an empty list? Or an empty map?

Reply:
Yes, that is not trivial, we could check for all cases but that is a bit verbose. I think it will fail eventually, but this is meant as a reminder that a credentialSubject is required. So if you provide a `"credentialSubject": []`. This check doesn't fail, but you can say that is was a conscious decision.